.github/workflows/linux.yml

```diff
@@ -642,6 +642,7 @@ jobs:
       INSTALL_DIR: ${{ github.workspace }}/ov
       SRC_DIR: ${{ github.workspace }}/src
       BUILD_DIR: ${{ github.workspace }}/build
+      OPENVINO_LOG_LEVEL: 1 # Debugging information doesn't guarantee stable results
```

Comment:
not sure that we need it. Is there any strong reason? I think we have enough logs now

Reply:
The debug logging includes messages about the time it takes to read the model. These messages differ from run to run, so we cannot compare the results of the CPP, Python, and JavaScript parts.

However, if this parameter is required for other sample tests, we can disable it for each GGUF test separately.